lib/jasmine-reportportal-reporter.js

```diff
@@ -17,21 +17,35 @@ class ReportportalReporter {
         this.parentIds = [];
         this.conf = conf;
         this.registerListeners();
+        this.reportHooks();
         this.additionalCustomParams = {};
         this.suiteDescription = new Map();
         this.suiteLogs = new Map();
         this.suiteAttributes = new Map();
+        this.hookIds = new Map();
+        this.startTime = null;
+    }
+
+    reportHooks() {
+        this.conf.reportHooks && this.installHooks();
     }
 
     escapeMarkdown(string) {
         return string.replace(/_/gm, '\\_').replace(/\*/gm, '\\*');
     }
 
-    getParentId() {
+    getParentId(number) {
```

Comment:
Please, give meaningful name to number. Or maybe extract logic getting parent of parent into separate method.